wordpress-mobile/WordPress-iOS
# Makes private and improves animatedBox in AbstractPostListViewControl.

### Description:
- Improves `animatedBox` in `AbstractPostListViewController` by making it non-optional, and by making it animate.
- Makes `animatedBox` private in `AbstractPostListViewController`.
- Fixes a curious bug in `WPAnimatedBox.m`.

### Additional info:

The way in which `WPAnimatedBox` is being animated now is kinda hackish - since the delay we added is not guaranteed to work.  A refactor to this classes would be better.

However, since the animation not working would be low-impact, I think the solution should be acceptable until I can get around to actually refactoring these classes.

### To test:
1. Launch the app
2. Go to the list of blogs and pick one
3. Open to the list of pages or posts
4. Pull down the list to refresh and make sure that while loading posts the 

/cc @frosty for review
/fyi @aerych (cos you know this code better).
